README.rst

```diff
@@ -0,0 +1,376 @@
+be involved in mail delivery, and can run on completely different machines to
+where your mailbox actually is. So this is the perfect tool to take good care
+of your ISP mailbox without having to leave it.
+
+You can read the full documentation in `Read the docs`_.
+
+.. _Read the docs: https://isbg.readthedocs.io/
+
+.. only:: html
+
+    .. contents:: Table of Contents
+       :depth: 3
+
+Features
+--------
+
+-  Works with all common IMAP servers
+-  Works on Linux, MacOS X and Windows (even smartphones!)
+-  Can do IMAP over SSL
+-  Can remember your password
+-  Will work painlessly against multiple IMAP accounts and servers
+-  Is not involved in the mail delivery process, and so can run on any
+   machine
+   that can contact your IMAP server
+-  Highly configurable
+-  Sensible defaults so you don't have to do any configuring :-)
+-  Compatibility with Python 2.7, 3.5, 3.6
+-  Possibility to skip spam detection to stick only to the teach feature
+-  Don't fail when meeting horrible and bad formed mail
+-  Lock file to prevent multiple instance to run at the same time (for
+   cron jobs)
+
+
+Installation
+------------
+
+**isbg** install a python package module and also a script to use it ``isbg``,
+it also install another script to unwrap messages: ``isbg_sa_unwrap``.
+
+Dependencies
+~~~~~~~~~~~~
+
+**isbg** is written in the Python language. Python is installed by default on
+most Linux systems. You can can find out more about Python at
+`Python home page`_.
+
+Make sure you have *SpamAssassin* installed. All the necessary information
+can be found on the `SpamAssassin wiki`_. *SpamAssassin* should be on your
+``$PATH`` (it installs in ``/usr/bin/`` by default)
+
+To run, ``isbg`` also depends on some python modules.
+
+- `docopt`_ for command line options.
+
+- `cchardet`_ or `chardet`_  for encoding detection.
+
+- `xdg`_ to found the ``.cache`` directory. ``xdg`` is not required, if it's
+  not installed, **isbg** will try to found ``.cache``.
+
+.. _Python home page: https://www.python.org/
+.. _SpamAssassin wiki: https://wiki.apache.org/spamassassin/FrontPage
+.. _docopt: https://pypi.python.org/pypi/docopt
+.. _cchardet: https://pypi.python.org/pypi/cchardet
+.. _chardet: https://pypi.python.org/pypi/chardet
+.. _xdg: https://pypi.python.org/pypi/xdg
+
+
+Install from source
+~~~~~~~~~~~~~~~~~~~
+
+From the main directory where you have download isbg, run::
+
+    $ python setup.py install --record installed_files.txt
+
+It will install under ``/usr/local/``. Supposing that you are using Python 2.7,
+the package module should be in ``/usr/local/lib/python2.7/dist-packages/`` and
+the main script will be ``/usr/local/bin/isbg``.
+
+In ``installed_files.txt`` there should be the list of files installed. To
+uninstall them, use::
+
+    $ tr '\n' '\0' < installed_files.txt | xargs -0 rm -vf --
+
+In windows systems, you can build a windows installer using::
+
+    $ python setup.py bdist_wininst
```

Comment:
Have you tested the Windows support? Are you willing to keep testing for it to support it?

We've been thinking about deprecating it for a while since neither @duk3luk3 nor me run Windows and want to test for it. 

More in #51